src/main/java/hellos/Hellos.java

```diff
@@ -30,6 +35,7 @@ private String andreasSaysHello() {
 
   private String patSaysHello() {
     return "Pat says 'Hello to everyone!'\n";
+
   }
```

Comment:
We typically wouldn't want a blank line like this at the end of a function definition.

```suggestion
  private String patSaysHello() {
    return "Pat says 'Hello to everyone!'\n";
  }
```